subgraphs/stakedao/src/Oracle/routers/CurveRouter.ts

```diff
@@ -0,0 +1,141 @@
+import * as utils from "../common/utils";
+import { getPriceUsdc } from "./SushiSwapRouter";
+import * as constants from "../common/constants";
+import { Address, BigDecimal, BigInt, dataSource } from "@graphprotocol/graph-ts";
+import { CurveRegistry as CurveRegistryContract } from "../../../generated/Controller/CurveRegistry";
+
+export function getCurvePriceUsdc(

```

Comment:
Does this handle v2 pools (i.e. non-like tokens)? Not necessary for this iteration, but need to have it at some point

Reply:
This is completely mimicking a oracle, so it should also work on v2 Pools.